pymc3/distributions/distribution.py

```diff
@@ -88,8 +88,11 @@ def getattr_value(self, val):
         return val
 
     def _repr_latex_(self, name=None, dist=None):
```

Comment:
Makes sense. You'd better make a simple property not to retype `__latex__ = ...` every time
```
@property
def __latex__(self):
    return self._repr_latex_
```